From: Julian Taylor <jtaylor.debian@googlemail.com>
Date: Mon, 12 Oct 2015 15:17:23 -0700
Subject: enable large file support

 probably not needed as the file open api of qhull is not available in python
Forwarded: no

Patch-Name: qhull-lfs.patch
---
 scipy/spatial/setup.py | 3 +++
 1 file changed, 3 insertions(+)

--- a/scipy/spatial/setup.py
+++ b/scipy/spatial/setup.py
@@ -52,6 +52,9 @@ def configuration(parent_package='', top
     config.add_extension('_qhull',
                          sources=['_qhull.c', 'qhull_misc.c'] + qhull_src,
                          **cfg)
+    cfg.setdefault('define_macros', []).append(('_FILE_OFFSET_BITS','64'))
+    cfg.setdefault('define_macros', []).append(('_LARGEFILE_SOURCE','1'))
+    cfg.setdefault('define_macros', []).append(('_LARGEFILE64_SOURCE','1'))
 
     # cKDTree
     ckdtree_src = ['query.cxx',
